documents/employment-contract-no.md

```diff
@@ -0,0 +1,128 @@
+# Ansettelseskontrakt
+
+## Arbeidsområde
+
+Arbeidets art er beskrevet i den ansattes rollebeskrivelse. Arbeidet skal utøves
+i samsvar med gjeldende formål og den til enhver tid vedtatt strategi for,
+innenfor rammen av de instrukser og anvisninger Bjerk AS' ledelse gir. Bjerk gir
+og forventer en betydlig grad av selvstendighet og initiativ innenfor rammeene
+av rollebeskrivelsen.
+
+Arbeidstakeren kan ikke uten skriftlig godkjennelse ha annet lønnet arbeid eller
+verv.
+
+## Plikter
+
+Under sitt arbeid skal arbeidstakeren sørge for at den virksomhet arbeidstakeren
+har ansvaret for og/eller selv utøver, drives i henhold til lover, forskrifter
+og etter god forretningsskikk. Arbeidstakeren skal særlig iaktta det forhold at
+Bjerk må opptre upartisk og uavhengig i markedet. Arbeidstaker må i alt sitt
+arbeid og i all sin fremferd sørge for at det ikke oppstår grunnlag for tvil om
+Bjerk AS' uavhengige og objektive stilling og dets absolutte pålitelighet.
+
+For øvrig plikter arbeidstakeren å følge gjeldende sikkerhetsprosedyrer og
+personalhåndbøker.
+
+## Arbeidsted
+
+Det faste arbeidsstedet avtales nærmere, men om ikke annet er avtalt er det ved
+tilstedeværelse i Bjerks lokaler.
+
+## Arbeidstid
+
+Normal arbeidstid er 40 timer per uke, inkl. 1⁄2 time lunsj pr. dag, med
+kjernetid fra 09.00 til 15.00. Arbeid utover normal arbeidstid kan i enkelte
+tilfeller forekomme om oppgavene ikke kan utføres tilfredsstillende innen normal
+arbeidstid. Overtidsreglene gjelder ikke for stillinger utenfor
+Arbeidsmiljølovens. § 10-12 om arbeidstid.
+
+Arbeidstaker plikter å gjennomføre pålagt arbeid innenfor gjeldende lover,
+offentlige vedtak og gyldige avtaler.
+
+Om ikke annet er avtalt, regnes alle stillinger i Bjerk som særlig uavhengig
+stillinger og er således unntatt Arbeidsmiljølovens. § 10-12 om arbeidstid.
+Dette begrunnes i at metoden Bjerk arbeider på innebærer en tydelig og åpenbar
+«selvstendighet» eller «uavhengighet» i hvordan og til hvilken tid
+arbeidsoppgaver organiseres og gjennomføres.
+
+Ansatte i Bjerk styrer sine egne arbeidsoppgaver; kundenes krav og omgivelser
+har styrer selvsagt dette noe.
+
+## Lønn
+
+Lønnen utbetales den 15. i hver måned i 11 måneden. I juni utbetales
+feriepenger. Lønn og andre vilkår skal være gjenstand for vurdering etter 3
+måneder i stillingen. Tid utover normal arbeidstid kompenseres ikke, verken i
+form av avspasering eller penger.
```

Comment:
Åpner vi ikke for muligheter om særskilte avtaler om avspasering? F.eks. at noen vil arbeide ekstra en uke og så ta en langhelg neste uken?

Reply:
Avspassering, overtid og disse tingene er administrasjon, noe jeg mener vi på generelt basis har lyst på minst mulig av. Det er kjernen i liberaliseringen av arbeidskontrakten. Den enkelte ansatte styrer dette selv (se «Arbeidstid»-punktet).

I tillegg, mener jeg personlig at vi bør ha minst mulig insentiver til å jobbe utover normal arbeidstid. @seacurrent, noe du vil legge til?

Reply:
Så lenge man har en særlig uavhengig stilling er man fri til, under eget ansvar, å håndtere dette. La det stå slik det er